.travis.yml

```diff
@@ -16,14 +16,14 @@ matrix:
       rvm: system
 
 before_install:
+  - export TRAVIS_COMMIT=$(git rev-parse --verify -q HEAD)
```

Comment:
Whenever we push new commits into master, Github will auto update `+refs/pull/$TRAVIS_PULL_REQUEST/merge` to make PR merge into latest master commit. As result, Travis will download the new merge commit but fail to update `TRAVIS_COMMIT`. This is why we see `Cannot parse object TRAVIS_COMMIT` error, and it won't be solved by refetching.

So I override the value to point it to the `FETCH_HEAD` which is also the commit we want to test.